tools/pycafe/create_pycafe_links.py

```diff
@@ -44,13 +44,10 @@ def generate_link(directory: str, extra_requirements: Optional[list[str]] = None
 
     # Requirements
     requirements = "\n".join(
-        filter(
-            None,
-            [
-                f"{PYCAFE_URL}/gh/artifact/mckinsey/vizro/actions/runs/{RUN_ID}/pip/{WHL_FILE}",
-                *(extra_requirements or []),
-            ],
-        )
+        [
```

Comment:
@maxschulz-COL I just came across this while I was getting the right link for pycafe. Did I miss something or was this `filter` not doing anything?

Reply:
Ah yes, you can skip it, the `or` statement fixes this. I was thinking we need it for the `None` cases, but that's not the case